### PR TITLE
LPD-27494 Remove classNameId cache for CTStore

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-27494

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-store-service/src/main/java/com/liferay/change/tracking/store/internal/CTStoreCTEventListener.java [hunk 1]
```diff
@@ -40,7 +41,7 @@ public class CTStoreCTEventListener implements CTEventListener {
 	@Override
 	public void onAfterPublish(long ctCollectionId) throws CTEventException {
 		List<CTEntry> ctEntries = _ctEntryLocalService.getCTEntries(
-			ctCollectionId, _ctsContentClassNameId);
+			ctCollectionId, _ctsContentClassNameId.get());
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-store-service/src/main/java/com/liferay/change/tracking/store/internal/CTStoreCTEventListener.java:44]: Use `portal.getClassNameId`

File: modules/apps/change-tracking/change-tracking-store-service/src/main/java/com/liferay/change/tracking/store/internal/CTStoreCTEventListener.java [hunk 2]
```diff
@@ -126,7 +127,7 @@ else if (_log.isWarnEnabled()) {
 	public void onBeforeRemove(long ctCollectionId) throws CTEventException {
 		List<Long> ctsContentIds =
 			_ctEntryLocalService.getExclusiveModelClassPKs(
-				ctCollectionId, _ctsContentClassNameId);
+				ctCollectionId, _ctsContentClassNameId.get());
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-store-service/src/main/java/com/liferay/change/tracking/store/internal/CTStoreCTEventListener.java:130]: Use `portal.getClassNameId`.

File: modules/apps/change-tracking/change-tracking-store-service/src/main/java/com/liferay/change/tracking/store/internal/CTStoreCTEventListener.java [hunk 3]
```diff
@@ -164,7 +165,7 @@ protected void activate(BundleContext bundleContext) {
 	@Reference
 	private CTEntryLocalService _ctEntryLocalService;
 
-	private long _ctsContentClassNameId;
+	private Supplier<Long> _ctsContentClassNameId;
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-store-service/src/main/java/com/liferay/change/tracking/store/internal/CTStoreCTEventListener.java:168]: Remove the variable.

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Comment by brookedalton** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**